src/nlp/dataset.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+import configparser
+from typing import NoReturn, Tuple
+from utils import preprocess_title
+from torch.utils.data import Dataset
+from transformers import BertTokenizer
+
+CONFIG = configparser.ConfigParser()
+CONFIG.read("nlp_config.ini")
+MODEL_NAME = CONFIG["model_configs"]["model_name"]
+MAX_LEN = CONFIG["dataset_configs"]["max_length"]
+TOKENIZER = BertTokenizer.from_pretrained(MODEL_NAME)
+
+
+class ShopeeNLPDataset(Dataset):
+    """
+    Dataset of preprocessed products titles
+    """
+
+    def __init__(self, path_to_file: str) -> NoReturn:
+        """
+        Args:
+            path_to_file (str): path to the dataset .csv file
+
+        Returns:
+            NoReturn
+        """
+
+        self.data = pd.read_csv(path_to_file)["title"].values
+
+    def __len__(self) -> int:
+        """Method for getting dataset length
+
+        Returns:
+            int: the length of the dataset
+        """
+
+        return len(self.data)
+
+    def __getitem__(self, index: int) -> Tuple[np.array, np.array]:
```

Comment:
Don't forget about built-in `tuple` for type hint

Reply:
Done